js/build.js

```diff
@@ -114,16 +103,21 @@ $('[data-login-ds-id]').each(function() {
       dataSourceId: data_source_id,
       where: where
     })
-      .catch(function (error) {
-        return Promise.reject(error);
-      });
+    .catch(function (error) {
+      return Promise.reject(error);
+    });
   }
 
   function resetFromDataSource(data_source_id, where) {
-    return Fliplet.DataSources.connect(data.dataSource, { offline: false })
-      .then(function(dataSource) {
-        return dataSource.sendValidation({type: 'email', where: where})
-      });
+    return Fliplet.DataSources.connect(data.dataSource, {
+      offline: false
+    })
+      .then(function (dataSource) {
+        return dataSource.sendValidation({
+          type: 'email',
+          where: where
+        })
+    });
```

Comment:
```suggestion
      });
```

...or move `.then()` to the previous line